tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -451,3 +451,248 @@ Feature: copy file
+    And the response when user "Alice" gets the info of the last share should include
+      | file_target | /Shares/BRIAN-FOLDER |
+
+
+  Scenario: copy a folder into another folder at different level which is received as a group share
+    Given using spaces DAV path
+    And group "grp1" has been created
+    And user "Alice" has been added to group "grp1"
+    And user "Brian" has been added to group "grp1"
+    And user "Brian" has created folder "BRIAN-FOLDER"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder/third-level-folder"
+    And user "Brian" has shared folder "BRIAN-FOLDER" with group "grp1"
+    And user "Alice" has accepted share "/BRIAN-FOLDER" offered by user "Brian"
+    And user "Alice" has created folder "Sample-Folder-A"
+    And user "Alice" has created folder "Sample-Folder-A/sample-folder-b"
+    And user "Alice" has created folder "Sample-Folder-A/sample-folder-b/sample-folder-c"
+    When user "Alice" copies folder "Sample-Folder-A/sample-folder-b" from space "Personal" to "/BRIAN-FOLDER/second-level-folder/third-level-folder" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And as "Alice" folder "/Sample-Folder-A/sample-folder-b/sample-folder-c" should exist
+    And for user "Alice" folder "BRIAN-FOLDER" of the space "Shares Jail" should contain these files:
+      | /second-level-folder/third-level-folder/sample-folder-c |
+    And the response when user "Alice" gets the info of the last share should include
+      | file_target | /Shares/BRIAN-FOLDER |
+
+
+  Scenario: copy a file into a folder at different level received as a group share
+    Given using spaces DAV path
+    And group "grp1" has been created
+    And user "Alice" has been added to group "grp1"
+    And user "Brian" has been added to group "grp1"
+    And user "Brian" has created folder "BRIAN-FOLDER"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder"
+    And user "Brian" has created folder "BRIAN-FOLDER/second-level-folder/third-level-folder"
+    And user "Brian" has shared folder "BRIAN-FOLDER" with group "grp1"
+    And user "Alice" has accepted share "/BRIAN-FOLDER" offered by user "Brian"
+    And user "Alice" has created folder "Sample-Folder-A"
+    And user "Alice" has created folder "Sample-Folder-A/sample-folder-b"
+    And user "Alice" has uploaded file with content "sample file-c" to "Sample-Folder-A/sample-folder-b/textfile-c.txt"
+    When user "Alice" copies folder "Sample-Folder-A/sample-folder-b/textfile-c.txt" from space "Personal" to "/BRIAN-FOLDER/second-level-folder" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Alice" folder "BRIAN-FOLDER" of the space "Shares Jail" should not contain these files:
+      | /second-level-folder/third-level-folder |
```

Comment:
check also content of that file, it should be `sample file-c`

Reply:
@individual-it  did you mean for file `second-level-folder` ? if so it is checked below.